partiql-spi/src/main/java/org/partiql/spi/Enum.java

```diff
@@ -3,16 +3,20 @@
 import java.util.Objects;
 
 /**
- * Represents an enum in the PartiQL Library. This is used for backward/forward compatibility purposes.
+ * All enumerated types should extend this class for backward/forward compatibility.
  */
-public abstract class PEnum {
+public abstract class Enum {
```

Comment:
Originally named `PEnum` due to the ubiquity of `java.lang.Enum`. I don't mind it either way since most people don't use `Enum` directly, but bringing it up for awareness.

Reply:
Yeah I figured since not used directly, parameterized, and requires explicit import then there's little to no ambiguity.